services/manufacturerService.js

```diff
@@ -4,40 +4,25 @@ const { errObject } = require("../helpers/helper")
 const Manufacturer = require("../models/manufacturerModel")
 
 exports.getPagedManufacturer = async (pageNo, docsPerPage) => {
-	try {
-		const totalDocs = await Manufacturer.countDocuments()
-		const manufacturers = await Manufacturer.find()
-			.skip(pageNo * docsPerPage)
-			.limit(docsPerPage)
-
-		const response = {
-			total: totalDocs,
-			manufacturers,
-		}
-		return response
-	} catch (error) {
-		throw error // Let the error handler handle it
+	const totalDocs = await Manufacturer.countDocuments()
+	const manufacturers = await Manufacturer.find()
+		.skip(pageNo * docsPerPage)
+		.limit(docsPerPage)
+
+	const response = {
+		total: totalDocs,
+		manufacturers,
 	}
+	return response
 }
 
 exports.createManufacturer = async (manufacturerData) => {
-	try {
-		const manufacturer = await Manufacturer.create(manufacturerData)
-		return manufacturer
-	} catch (error) {
-		// console.log(error)
-		throw error
-	}
+	const manufacturer = await Manufacturer.create(manufacturerData)
+	return manufacturer
```

Comment:
cant we just not have 
`return await Manufacturer.create(manufacturerData)`
why do we need to initialize the `manufacturer` variable